cumulusci/cli/cci.py

```diff
@@ -1639,3 +1634,14 @@ def gist(runtime):
     else:
         click.echo(f"Gist created: {gist.html_url}")
         webbrowser.open(gist.html_url)
+
+
+def group_items(items):
+    groups = {}
+    for item in items:
+        group = item["group"] or "Other"
+        if group not in groups:
+            groups[group] = []
```

Comment:
This would be a good place to use a collections.defaultdict instead of a plain dict